mcux/drivers/imx/fsl_common.h

```diff
@@ -38,8 +38,8 @@
 
 /*! @name Driver version */
 /*@{*/
-/*! @brief common driver version 2.0.1. */
-#define FSL_COMMON_DRIVER_VERSION (MAKE_VERSION(2, 1, 0))
+/*! @brief common driver version 2.2.0. */
+#define FSL_COMMON_DRIVER_VERSION (MAKE_VERSION(2, 2, 0))
```

Comment:
How are we getting a newer version of fsl_common.h and other drivers, when RT1010 has SDK 2.6.0 and RT1060 has SDK 2.6.2?

Reply:
RT1060 has several post RFP releases to add wifi support